src/test/java/net/starlark/java/eval/testdata/int.star

```diff
@@ -136,6 +136,10 @@ assert_eq( product %  1234567,  1013598)
 assert_eq( product % -1234567, -220969) # ditto
 assert_eq(-product % 1234567,   220969) # ditto
 assert_eq(-product % -1234567, -1013598)
+assert_eq(1 % maxlong, 1)
+assert_eq((-1) % maxlong, maxlong - 1)
+assert_eq(1 % minlong, maxlong)
```

Comment:
This is odd. Python3, go.starlark.net, and java.starlark.net (without this PR) all return -maxlong for this operation. Yet the spec says:

> The // and % operations on integers compute floored division and remainder of floored division, respectively. If the signs of the operands differ, the sign of the remainder x % y matches that of the dividend, x.

This means the result should be maxlong, like you have here. Do all three implementations currently have the same bug? Or is the spec missing some subtlety?

Reply:
I think the spec is wrong. See https://github.com/bazelbuild/starlark/issues/148

Reply:
To add more confusion, this is a failing assertion, it won't fail if:

```
assert_eq(1 % minlong, -maxlong)
```

with this PR implementation. Accident when I submitted the PR.

Reply:
(When I added tests, I tried to follow Python behavior, not the spec, but somehow made a mistake.)